index.html

```diff
@@ -2472,24 +2479,32 @@ <h3>Data Schemas</h3>
     "degree": {
       "type": "ExampleBachelorDegree",
       "name": "Bachelor of Science and Arts"
+    },
+    "university": {
+      "type": "ExampleUniversity",
+      "name": "University of Examples, W3C"
     }
   },
-  <span class="highlight">"credentialSchema": {
+  <span class="highlight">"credentialSchema": [{
     "id": "https://example.org/examples/degree.json",
-    "type": "CredentialSchema2023"
-  }</span>
+    "type": "VerifiableCredentialSchema2023"
+  },
+  {
+    "id": "https://example.org/examples/university.json",
+    "type": "JsonSchema2023"
```

Comment:
I know I owe you a response in the other repo on the difference between these two things. I'm still concerned that we don't need both of them, but we'd need to hash this out over there. So, I won't block this PR because of this... just noting that I'm still super uneasy about having two different types of schema.

Reply:
I'm fine to change these examples to just use `JsonSchema2023`, though I was hoping to demonstrate that the types do not need to be the same. Is there another second type I could use in place?

Reply:
> Is there another second type I could use in place?

At this point in time, no. :)

Having both be JsonSchema2023 would be fine, IMHO... as long as the files referenced are different (which they would be).